docs/supported-technologies.asciidoc

```diff
@@ -88,9 +88,9 @@ The Node.js agent will automatically instrument the following modules to give yo
 |https://www.npmjs.com/package/mongodb-core[mongodb-core] |>=1.2.19 <4 |Will instrument all queries.
 A lot of higher level MongoDB modules use mongodb-core,
 so those should be supported as well
-|https://www.npmjs.com/package/mongodb[mongodb] |>=2.0.0 |Supported via mongodb-core
-|https://www.npmjs.com/package/mongojs[mongojs] |>=1.0.0 |Supported via mongodb-core
-|https://www.npmjs.com/package/mongoose[mongoose] |>=4.0.0 |Supported via mongodb-core
+|https://www.npmjs.com/package/mongodb[mongodb] |>=2.0.0 <3.3.0 |Supported via mongodb-core
+|https://www.npmjs.com/package/mongojs[mongojs] |>=1.0.0 <2.7.0 |Supported via mongodb-core
```

Comment:
We still support all versions of `mongojs` as that version still relies on mongodb 2.x

```suggestion
|https://www.npmjs.com/package/mongojs[mongojs] |>=1.0.0 |Supported via mongodb-core
```

Reply:
Ups, I didn't see your comment https://github.com/elastic/apm-agent-nodejs/issues/1368#issuecomment-534327436 before after writing this review. Let's just keep your suggested max version 👍